src/main/java/racingcar/domain/Car.java

```diff
@@ -15,16 +15,16 @@ private Car() {
     }
 
     private Car(final String name) {
+        validateName(name);
         this.position = DEFAULT_DISTANCE;
         this.name = name;
     }
 
     public static Car of(final String name) {
-        validateName(name);
         return new Car(name);
     }
 
-    private static void validateName(String name) {
+    private static void validateName(final String name) {
```

Comment:
```suggestion
    private static void validate(final String name) {
```
요건 어떤가요? 만약 초기화 파라미터로 name말고 다른 문자열이 들어왔다고 해보면, 
동일한 일을 하는 함수를 더 만들게 되는데 🤔 

이름을 검증한다. 라는의미는 아래와 같이 해도 충분히 전달 될거 같아요. 어떤가요?
```
validate(name);
```